packages/modal/src/ModalHeader.js

```diff
@@ -16,43 +17,43 @@ const StyledCloseButton = styled(CloseButton)`
   }
 
   &:focus {
-    background-color: ${(props) => props.theme.palette.primary.dark};
+    box-shadow: none;
     outline: none;
   }
 `
 
-const ModalHeader = ({ bg, color, onClose, title }) => (
+const ModalHeader = ({ bg, color, onClose, textStyle, title, ...space }) => (
```

Comment:
`...space` is really all remaining props, should we rename this to `...rest` or `...props`?

Reply:
Updated!